Jenkinsfile

```diff
@@ -143,6 +145,47 @@ pipeline {
             }
           }
         }
+        stage('Windows') {
+          agent { label 'windows-2019-docker-immutable' }
```

Comment:
@jackshirazi this is the label that we should update to `windows-10` once https://github.com/elastic/infra/pull/31021 is there.

As @v1v is coming back next week, we can discuss about the worker types then

Reply:
@jackshirazi and @eyalkoren 

It appears that [we can't proceed with Docker on Windows 10 as originally planned](https://github.com/elastic/infra/issues/31282#issuecomment-973112476). We'll need to consider other approaches.

Reply:
thanks. looks like we'll have to go with windows server and disable the testcontainer tests for windows. Will discuss this in the team